src/scripts/main.js

```diff
@@ -2,7 +2,40 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  const tr = document.createElement('tr');
+
+  const tdN = document.createElement('td');
+
+  tdN.innerHTML = person.name;
+  tr.appendChild(tdN);
+
+  const tdG = document.createElement('td');
+
+  tdG.innerHTML = person.sex;
+  tr.appendChild(tdG);
+
+  const tdB = document.createElement('td');
+
+  tdB.innerHTML = person.born;
+  tr.appendChild(tdB);
+
+  const tdD = document.createElement('td');
+
+  tdD.innerText = person.died;
+  tr.appendChild(tdD);
+
+  const tdAge = document.createElement('td');
+
+  tdAge.innerText = person.died;
+  tr.appendChild(tdAge);
+
+  const tdC = document.createElement('td');
+
+  tdC.innerText = Math.ceil(person.died / 100);
+  tr.appendChild(tdC);
```

Comment:
```suggestion
  tr.apend(tdA, tdB, tdC ...)
```
Можно сразу все в одной строке зааппендить